docs/lint_errors.md

```diff
@@ -227,14 +195,11 @@ The `README.md` files for a project are very important and must meet some requir
 
 ## Error #7 - Pipeline and container version numbers ## {#7}
 
-> This test only runs when `--release` is set or `$TRAVIS_BRANCH` is equal to `master`
+> This test only runs when `--release` is set
```

Comment:
We need to mirror this functionality to GitHub Actions and update the docs here, rather than deleting it.

Code where this is detected:
https://github.com/nf-core/tools/blob/9e33e7f02758964a1a09436af1eb6afe0e596e4c/scripts/nf-core#L232

This means that when the tests are running on the `master` branch of the nf-core repo, it assumes that the code is primed for a release and runs the lint tests with the additional `--release` option which makes it more strict in places.

Reply:
https://help.github.com/en/actions/automating-your-workflow-with-github-actions/using-environment-variables#default-environment-variables

I guess we want `GITHUB_REPOSITORY ` and `GITHUB_REF`

Reply:
See https://github.com/nf-core/tools/pull/502/commits/d7c06b61e7abfef8298c63bef0d9817e72949bb3

Reply:
See https://github.com/nf-core/tools/pull/502/commits/7b597877656b0d306253b904bac3d15d37de1c43

Reply:
See https://github.com/nf-core/tools/pull/502/commits/5cc52b9ebd77019cc1fa7ea018c054e1f752300e